src/mod_last_mnesia.erl

```diff
@@ -69,6 +70,11 @@ remove_user(_HostType, LUser, LServer) ->
     F = fun() -> mnesia:delete({last_activity, US}) end,
     wrap_transaction_result(mnesia:transaction(F)).
 
+% Implementation only for RDBMS backends
+-spec remove_domain(host_type(), jid:lserver()) -> ok.
+remove_domain(_HostType, _Domain) ->
+    ok.
```

Comment:
do we need call eg. `remove_user/3` for delete user by domain or create a new function for to do it?

Reply:
We discussed this kind of iterative approach and came to the conclusion that it could be too slow for `mnesia` and decided to skip backends other than RDBMS for the moment.

Reply:
We were skipping implementation of domain removal for other backends than RDBMS for now :)